makefile.cargo

```diff
@@ -39,7 +39,6 @@ CXXFLAGS += \
 	-std=gnu++11 \
 	-fPIC \
 	-Ilibazure \
-	-I/usr/include/freetype2 \
```

Comment:
Yes, please.  The other changes should be harmless, and we'll need them for Linux on ARM.